configuration/pih/htmlforms/ncd-adult-followup.xml

```diff
@@ -821,13 +808,11 @@
                                                          answerLabels="Na,K+,Pro,Kcal"/>
                                                 </span>
                                             </td>
-                                        </obsgroup>
                                     </tr>
                                 </includeIf>
 
                                 <includeIf velocityTest="$pt_has_lf">
                                     <tr>
-                                        <obsgroup groupingConceptId="PIH:HEPATITIS C TEST - QUALITATIVE">
```

Comment:
This section won't work.  You can either add a construct/obsgroup OR add a dateLabel attribute to the Hep C test obs

dateLabel: Text to display after the obs.value widget and before the obs.datetime widget (obs.obs_datetime). Specifying this implies showDate="true".
OPTIONAL
Example: <obs conceptId="5497" labelText="Most recent CD4:" dateLabel="Date of test:"/>

Reply:
@lnball Thanks for the feedback

Reply:
@lnball Kindly double-check my update. Hope am correct!